source/Halibut.TestUtils.CompatBinary.Base/Services/StayAliveUntilHelper.cs

```diff
@@ -1,17 +1,16 @@
 using System;
 using System.IO;
-using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Halibut.TestUtils.SampleProgram.Base.Services
 {
     public class StayAliveUntilHelper
     {
-        public static async Task WaitUntilSignaledToDie()
```

Comment:
We probably never want to cancel this, the sub process should never continue if the parent dies.